### PR TITLE
Add responsive desktop upgrade files

### PR DESCRIPTION
**Pull request by diegoh40**
In this part of the project we used the responsive UXpart to elevate the project from a mobile device to a desktop environment, implemented several concepts learned during the week such as CSS media quaries, and CSS grid layout concepts, Flexbox, in addition to handling pair coding, and developed code review skills in this exercise.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,32 +4,50 @@
...
       <header class="mobile-header">
         <a href="#">Diego Hernández</a>
-        <a href="#"><img src="images2/menu.png" alt="hamburger Icon" /></a>
+                
+      <ul class="menu-deployed">
+          <li>Portfolio</li>
+          <li>About</li>
+          <li>Contact</li>
+          <li class="icon">
+          <img src="images2/menu.png" alt="hamburger Icon" />
```
Comment (OLIPLICHE) [inline index.html:28]: - [ ] Please fix the hamburger icon.
- Your design 
h

File: index.html [hunk 2]
```diff
@@ -46,14 +64,24 @@ <h3 class="subtitle">I’am web a developer.</h3>
...
         </ul>
+
+      </div>
+    </div>
       </section>
       <section class="my-works1">
         <h1 class="blacktitle">My Recent works</h1>
+        <div class="line1">
+          <hr>
+          </div>
```
Comment (OLIPLICHE) [inline index.html:75]: - [ ] Please the title "My Recent Works" is aligned to the left and after there's a line that **takes the rest of the page's width**. Kindly adjust your design.
- Your design:
la
- Original Design:
lb

File: index.html [hunk 3]
```diff
@@ -66,24 +94,45 @@ <h3>Multi-Post Stories</h3>
           <li>boostrap</li>
           <li>ruby</li>
         </ul>
-
+        <div class="button_works">
         <button type="submit">See Project</button>
```
Comment (OLIPLICHE) [inline index.html:98]: - [ ] Please the `See Project` button takes you to the GitHub repository. Kindly link that.

File: index.html [hunk 4]
```diff
@@ -94,17 +143,121 @@ <h3>Professional Art Printing Data</h3>
...
+          <li>bootstrap</li>
+          <li>ruby</li>
+        </ul>
+      </div>
+
+        <button  type="submit">See Project</button>
+      </div>
+    
+
+      <div class="bg-page-two">
```
Comment (OLIPLICHE) [inline index.html:177]: - [ ] Please when the project is hovered on with the mouse the description and dark opacity disappears and a `See Project` button appears to the bottom of the image and takes the whole image width

File: index.html [hunk 5]
```diff
@@ -94,17 +143,121 @@ <h3>Professional Art Printing Data</h3>
...
+    </div>
+      
 
       <section class="about">
-        <div class="about-me">
-          <div></div>
+        <div class="about-me-header">
+         
           <h1>About me</h1>
           <p>
```
Comment (OLIPLICHE) [inline index.html:262]: - [ ] Please there are 4 to 5 lines of text describing the student's experience. I suggest that you assign a "width: 100%" to `<p>` to adjust the text a little bit

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @diegoh40 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linter is green (the last commit have a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
- Github flow is correctly used (Pull Request is made from the feature branch into the main branch) 👏
- Pull Request is descriptive 👏

## Required Changes ♻️

- [ ] Please add Pull Request that have a descriptive "title" telling you what changes are inside

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by diegoh40** [comment]
I have corrected the errors suggested by the code reviewer, mostly of layout, were successfully corrected, understanding the concepts of proper configuration in margins paddings and border indicated at this stage of the project, so that the XU making it as similar as possible to the template # 2 of Figma.

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @diegoh40   , 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Github flow used properly.
:heavy_check_mark: Nice PR.

### Optional suggestions :arrow_down_small: 

- [ ] In your `README` file, Kindly Update the content for the getting started section. This should be an example of how you may give instructions on setting up your project locally.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**